app/javascript/components/close_contact/CloseContact.js

```diff
@@ -132,17 +169,67 @@ class CloseContact extends React.Component {
             </Row>
             <Row>
               <Form.Group as={Col} controlId="email">
-                <Form.Label className="nav-input-label">Email</Form.Label>
+                <Form.Label className="nav-input-label">Email {schema?.fields?.email?._exclusive?.required && '*'} </Form.Label>
                 <Form.Control size="lg" className="form-square" value={this.state.email || ''} onChange={this.handleChange} />
                 <Form.Control.Feedback className="d-block" type="invalid">
                   {this.state.errors['email']}
                 </Form.Control.Feedback>
               </Form.Group>
             </Row>
+            <Row>
+              <Form.Group as={Col} controlId="last_date_of_exposure">
+                <Form.Label className="nav-input-label">Last Date of Exposure {schema?.fields?.last_date_of_exposure?._exclusive?.required && '*'}</Form.Label>
+                <DateInput
+                  id="last_date_of_exposure"
+                  date={this.state.last_date_of_exposure}
+                  minDate={'2020-01-01'}
+                  maxDate={moment().format('YYYY-MM-DD')}
+                  onChange={this.handleDateChange}
+                  placement="top"
+                  isInvalid={!!this.state.errors['last_date_of_exposure']}
+                  customClass="form-control-lg"
+                  ariaLabel="Last Date of Exposure Input"
+                />
+                <Form.Control.Feedback className="d-block" type="invalid">
+                  {this.state.errors['last_date_of_exposure']}
+                </Form.Control.Feedback>
+              </Form.Group>
+            </Row>
+            <Row>
+              <Form.Group as={Col} className="mb-2 pt-2" controlId="assigned_user">
+                <Form.Label className="nav-input-label">
+                  Assigned User {schema?.fields?.assigned_user?._exclusive?.required && '*'}
+                  <InfoTooltip tooltipTextKey="assignedUser" location="top"></InfoTooltip>
```

Comment:
Any reason we added a tooltip for the Assigned User field but not the Last Date of Exposure field?

Reply:
To keep it consistent with the rest of the App. I copy and pasted the Assigned User code from the Enrollment. The `<infotooltip/>` for LDE didnt make sense for close contact, so I didnt copy it over. Happy to if we want it though.

Reply:
We'll see how folks feel in UAT.